server/game/cards/10-SoD/MaesterKedry.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../drawcard.js');
+
+class MaesterKedry extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Reduce next event by 2',
+            cost: ability.costs.kneelSelf(),
+            handler: context => {
+                this.untilEndOfPhase(ability => ({
+                    targetType: 'player',
+                    targetController: 'current',
+                    effect: ability.effects.reduceNextPlayedCardCost(2, card => card.getType() === 'event')
```

Comment:
Checking the card type should (theoretically) be redundant since only events can be 'played'. Is there a situation I'm not thinking of or a general bug in the reducer system?

Reply:
Not that I know of, should be good.